cmd/kube-controller-manager/app/options/options.go

```diff
@@ -53,6 +53,7 @@ func NewCMServer() *CMServer {
 			ConcurrentResourceQuotaSyncs:      5,
 			ConcurrentDeploymentSyncs:         5,
 			ConcurrentNamespaceSyncs:          2,
+			ConcurrentSATokenSyncs:            5,
```

Comment:
I guess that would be inconsistent and I won't ask you to change them all. Sigh. "Syncs" is confusing.